### DISCUSSION TITLE
5.3.0

### DISCUSSION THREAD

**Discussion by kobylynskyi** [kobylynskyi/graphql-java-codegen#795] [🚀 Releases]
* Support RegEx in `customAnnotationsMapping` #783 #793 @kobylynskyi
* Add @JvmStatic annotation to Kotlin builders #771 @kbrooks
* Support mapping of enum values in directive parameters #673 #674 @meistermeier
* Scala: Avoid duplicates in enum imports #674 @meistermeier
* Scala: Fix `generateModelOpenClasses` #741 @jxnu-liguobin

<hr /><em>This discussion was created from the release <a href='https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v5.3.0'>5.3.0</a>.</em>

**Comment by kobylynskyi** [comment]
@jxnu-liguobin could you please release SBT plugin? Thanks!

**Reply by jxnu-liguobin** [reply]
It will be released soon.
